### ISSUE TITLE
Gaia TAP+ documentation out of date?

### ISSUE THREAD

**Issue by AllenDowney** [astropy/astroquery#1756]
I was just working through the examples at

https://astroquery.readthedocs.io/en/v0.3.9/gaia/gaia.html

and it looks like some parts of the API might have changed.

Specifically, in this example:

```
>>> for column in (gaiadr1_table.get_columns()):
>>>   print(column.get_name())
```

It looks like the `get_columns()` method has been replaced with a `columns` property and `get_name()` has been replaced with `name`

So this works for me:

```
for column in table.columns:
    print(column.name)
```

**Comment by keflavich** [comment]
Thanks @AllenDowney.  If I'm reading this right, it's a documentation issue - we'd welcome a pull request to fix it!

**Comment by AllenDowney** [comment]
Done: https://github.com/astropy/astroquery/pull/1757

**Comment by keflavich** [comment]
Solved by #1757
